.github/workflows/nitpicks.yml

```diff
@@ -0,0 +1,80 @@
+---
+name: Nitpicks
+
+on:
+  pull_request:
+    paths:
+        - 'DC-*'
+        - 'xml/*'
+        - 'adoc/*'
+
+jobs:
+  nitpicks:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Checking out relevant branches
+        run: |
+          git checkout $GITHUB_BASE_REF || { echo "There's a Git issue"; exit 1; }
+          git checkout $GITHUB_HEAD_REF || { echo "There's a Git issue"; exit 1; }
+
+      - name: Checking for tabs
+        run: |
+          tab=$(git diff -U0 $GITHUB_BASE_REF..$GITHUB_HEAD_REF | sed -n '/^+/ p' | sed -n '/\t/ p' | sed -r -e 's/\t/→/g')
+          if [[ -n "$tab" ]]; then
+            echo -e "\nThis pull request introduces tabs (→) on the following lines:"
+            echo -e "\n$tab\n"
```

Comment:
> An exit command isn't really necessary if you use the template from previous item. The Action will stop this step anyway.

If that is true, then using two echos with `::error::` in succession as suggested here won't work, because it'd exit after the first `::error::` message. I am also unsure if `::error::` allows for `-e` which is extremely necessary for the second `echo`.

> Additionally, an exit just blows up the code for my taste.

`exit 1;` is 7 characters ... if anything "blows up the code", it's hardly that.^^

> You can use the continue-on-error: true. What you need is to "summarize" the result of each steps into an output. Then in a last step you check each one and react accordingly

I should probably do something like that. It could have been less work though if continue-on-error worked better though. :/

Reply:
> > An exit command isn't really necessary if you use the template from previous item. The Action will stop this step anyway.
> 
> If that is true, then using two echos with `::error::` in succession as suggested here won't work, because it'd exit after the first `::error::` message. I am also unsure if `::error::` allows for `-e` which is extremely necessary for the second `echo`.

Right, maybe I misremembered that. 

> >  Additionally, an exit just blows up the code for my taste.
>
> exit 1; is 7 characters ... if anything "blows up the code", it's hardly that.^^

It's not only this line, there are other lines. But it's probably a matter of taste, so ignore that. :wink: 

> I should probably do something like that. It could have been less work though if continue-on-error worked better though. :/

Right. It was just an idea. As you wrote, maybe for the future it would be better anyway to move it into a "real" GH Action. But for the time being, you could leave it as is.